keyboards/dm9records/tartan/keymaps/via/config.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 padfoot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
```

Comment:
Given the size of the matrix, this may be pushing the limits of the EEPROM, and at the expense of dynamic EEPROM macros, too.

Reply:
Yes. While other smaller boards were able to support more layers, this one maxed out at 6. But I have verified that this works and the more the merrier for VIA users. So I chose to have it as 6. Thanks for reviewing and approving!